mapf-viz/src/visibility_visual.rs

```diff
@@ -150,12 +150,13 @@ impl<Message, G: Grid> VisibilityVisual<Message, G> {
     fn find_closest(&self, p: iced::Point) -> Option<Cell> {
         let mut closest: Option<(Cell, f64)> = None;
         let r = self.visibility.agent_radius();
+        let r_squared = r * r;
         let p = Point::new(p.x as f64, p.y as f64);
 
         for (cell, _) in self.visibility.iter_points() {
             let p_cell = cell.center_point(self.grid().cell_size());
-            let dist = (p_cell - p).norm();
-            if dist <= r {
+            let dist = (p_cell - p).norm_squared();
```

Comment:
Very small nitpick: let's change the variable name to `dist_squared` so that readers who jump to the following comparison don't think the comparison is flawed.

Reply:
Agreed.